### ISSUE TITLE
KITTI-Caltech preprocessed data & training details

### ISSUE THREAD

**Issue by Kay1794** [gaozhangyang/SimVP-Simpler-yet-Better-Video-Prediction#5]
Thank you for sharing the code of this interesting paper. I notice you have shown the evaluation on the Caltech dataset, I am wondering if it is possible to also share your preprocessed KITTI-Caltech dataset? I assume you use the KITTI training set from directly from PredNet, whose resolution is 128 X 160, if that is the case, then only the Caltech test set will be enough. 

Since I am trying to replicate the experiment for SimVP on the KITTI-Caltech setting, some training details of your experiment would also be very helpful, e.g. learning rate, learning scheduler, and batch size. 

Thank you so much for your time!

**Comment by gaozhangyang** [comment]
Thank you for your interest. We plan to release these information in SimVP V2, which is coming soon.
